docs/scripts_md/ExitCodes.md

```diff
@@ -0,0 +1,74 @@
+# NAME
+
+NeuroDB::ExitCodes -- Class regrouping all the exit codes used by the imaging
+insertion scripts
+
+# SYNOPSIS
+
+    use NeuroDB::ExitCodes;
+
+    # testing if an argument was given to the script
+    if ( !$ARGV[0] ) {
+        print $Help;
+        print "$Usage\n\tERROR: Missing argument\n\n";
+        exit $NeuroDB::ExitCodes::MISSING_ARG;
+    }
+
+    # if script ran successfuly, exit with success exit code (a.k.a. 0)
+    exit $NeuroDB::ExitCodes::SUCCESS;
+
+# DESCRIPTION
+
+This class lists all the exit codes used by the imaging insertion scripts.
+
+The exit codes are organized per script, allowing 10 exit codes per script
```

Comment:
you cna say they are organized per script, together with a section that represents common failures to most scripts.

Reply:
👍